README.md

```diff
@@ -82,504 +63,41 @@ version. A copy of the GNU General Public License can be found in the
-as the one that was used to build XSPEC itself. The `gfortran_libraries`
-option, in particular, can be used on Linux to set the specific name of
-the library, e.g. `:libgfortran.so.3`.
-
-In order for the module to work, the `HEADAS` environment variable
-**must** be set in the shell from which the Python session is started.
-
-In order to check that the module is working, importing the
-`sherpa.astro.ui` module will no-longer warn you that the
-`sherpa.astro.xspec` module is not available and you can use routines
-such as:
-
-    % python -c 'from sherpa.astro import xspec; print(xspec.get_xsversion())'
-    12.10.0e
-
-Other customization options
----------------------------
-
-Sherpa supports other build configuration options that are required to
-support Sherpa build in specific environments, for instance when
-building the [CIAO analysis system](http://cxc.harvard.edu/ciao/). 
-These options include:
-
-- building Sherpa against a local version of the `CIAO` `region` and `group`
-  libraries
-- specify additional `CFLAGS` options for the group library
-- building Sherpa against a local build of the `wcssubs` routines
-- change the default `./configure` command line
-
-The `setup.cfg` file in the Sherpa source distribution contains more information
-about these options.
-
-Building the documentation
-==========================
-
-The Sphinx documentation is available in the `docs/` directory. It is designed
-so that the documentation can be built without needing to build (or install)
-Sherpa, but this requires Python 3.5 or higher. An example for setting
-up the build environment is:
-
-    % conda create -n=sherpa-sphinx python=3.5 'sphinx >= 1.3' sphinx_rtd_theme
-    % source activate sherpa-sphinx
-
-Using the build_sphinx target
------------------------------
-
-    % python setup.py build_sphinx
-
-The location of this output depends on the version os Sphinx in use. With
-version 1.4 it appears to be located at `build/sphinx/html/index.html`.
-
-Using the sphinx-build tool
----------------------------
-
-The following will create the documentation in `docs/build/html/index.html`.
-
-    % cd docs
-    % sphinx-build -b html . build/html
-
-Testing the documentation with Travis
--------------------------------------
-
-There is a documentation build included as part of the Travis-CI test suite,
-but it is not set up to do much validation. That is, you need to do something
-quite severe to break this build. Please see
-[issue 491](https://github.com/sherpa/sherpa/issues/491)
-for more information.
+Source installation is available for platforms incompatible with the
+binary builds, or for when the default build options are not sufficient
+(such as including support for the
+[`XSPEC` model library](https://heasarc.gsfc.nasa.gov/xanadu/xspec/)).
+The steps are described in the
+[building from source](https://sherpa.readthedocs.io/en/latest/install.html#building-from-source)
+documentation.
 
 History
```

Comment:
I believe we are missing a couple of Zenodo DOIs for the latest versions. That's something else I can take care of, and it's just easier to do it in this PR.